test/integration/targets/iam_user/tasks/main.yml

```diff
@@ -104,6 +131,155 @@
         - iam_user_info.iam_users[0].user_id == test_iam_user.user_id
         - iam_user_info.iam_users[0].user_name == test_iam_user.user_name
 
+  # ===========================================
+  # Test Managed Policy management
+  #
+  # Use a couple of benign policies for testing:
+  # - AWSOrganizationsReadOnlyAccess
+  # - ServiceQuotasReadOnlyAccess
+  #
+  - name: attach managed policy to user (check mode)
+    check_mode: yes
+    iam_user:
+      name: '{{ test_user }}'
+      state: present
+      managed_policy:
+        - arn:aws:iam::aws:policy/AWSOrganizationsReadOnlyAccess
+    register: iam_user
+
+  - name: assert that the user is changed
+    assert:
+      that:
+        - iam_user is changed
+
+  - name: attach managed policy to user
+    iam_user:
+      name: '{{ test_user }}'
+      state: present
+      managed_policy:
+        - arn:aws:iam::aws:policy/AWSOrganizationsReadOnlyAccess
```

Comment:
We also support referring to these as just 'AWSOrganizationsReadOnlyAccess' so we should probably test this behaviour